mq/3/on-premise/management/index.md

```diff
@@ -0,0 +1,196 @@
+---
+title: IronMQ On-Premise Management
+summary: "Management information for IronMQ clusters"
+layout: default
+section: mq-on-premise
+---
+
+<p class="subtitle">
+<!--   IronMQ On-Premise administration -->
+</p>
+
+<section id="toc">
+<h3>Administration</h3>
+<ul>
+<li><a href="#stats">Stats endpoint</a></li>
+<li><a href="#troubleshooting">Troubleshooting</a></li>
+<li><a href="#backups">Backups</a></li>
+</ul>
+</section>
+
+<h2 id="stats">Statistics</h2>
+
+You can get statistics from an IronMQ instance by using the endpoint:
+
+`GET /3/stats`
+
+You'll have to specify an admin token either in the query string `oauth` or in
+the Authorization headers, `Authorization: OAuth $token`.
+
+Response will be up to the last 60 intervals of stats, where interval can be set in the
+config (default: 60 seconds), at line: `"stats":{"interval":60}`.
+
+```
+{
+  "stat_history": [
+  {
+    "stats": {
+      "reserved": {
+        "requests": 22,
+        "messages": 22,
+        "avg_latency_ms": 38.416678136363636
+      },
+      "totals": {
+        "requests":42,
+        "messages":42,
+        "avg_latency_ms": 42.00
+      }
+    },
+    "start_time": "2015-02-27T17:17:33.11477666Z",
+    "end_time": "2015-02-27T17:18:33.114765082Z"
+  },
+  ...
+  ]
+}
+```
+
+<h2 id="troubleshooting"></h2>
+
+# IronMQ Enterprise Troubleshooting Guide
+
```

Comment:
should we mention the canonical 'have you tried turning it off and on again' solution?